x-pack/lib/filters/geoip/database_manager.rb

```diff
@@ -26,255 +24,107 @@
 #   while `offline` is for static database path provided by users
 
 module LogStash module Filters module Geoip class DatabaseManager
-  extend LogStash::Filters::Geoip::Util
   include LogStash::Util::Loggable
-  include LogStash::Filters::Geoip::Util
   include Singleton
 
+  DB_EXT = 'mmdb'.freeze
+  GEOLITE = 'GeoLite2-'.freeze
+
+  CITY = "City".freeze
+  ASN = "ASN".freeze
+  DB_TYPES = [ASN, CITY].freeze
+
+  CC = "CC".freeze
```

Comment:
This one got tricky: the pre-existing specs _also_ declared `CITY` and `ASN` etc. as references to _these_ constants without namespacing them, so the specs used global constants that weren't available to runtime code, and with the refactor these constants were no longer referenced otherwise (or referenced a single time).

I've inlined the single-use ones, removed the unreferenced ones, and made some small changes to the specs to just use the string values. Including a module into an example group does _not_ make that module's constants available, and I figured this was the path of least resistance. I did consider using symbols instead of strings (safely handling strings if-and-only-if they are already in the list of symbols), but the scope of that _feels_ like a bigger change than we need.